lms/resources/_js_config/__init__.py

```diff
@@ -215,6 +215,12 @@ def enable_content_item_selection_mode(self, form_action, form_fields):
                     "formAction": form_action,
                     "formFields": form_fields,
                     "createAssignmentAPI": self._create_assignment_api(),
+                    # The "content item selection" that we submit to Canvas's
+                    # content_item_return_url is actually an LTI launch URL with
+                    # the selected document URL or file_id as a query parameter. To
+                    # construct these launch URLs our JavaScript code needs the
+                    # base URL of our LTI launch endpoint.
+                    "ltiLaunchUrl": self._request.route_url("lti_launches"),
```

Comment:
Does this mean we now have `ltiLaunchUrl` in multiple places?  I think it's still in `FilePickerConfig.canvas_config()` as well